scripts/import/import_gene_disease.py

```diff
@@ -248,70 +557,133 @@ def insert_mondo_gene_diseases(db_host, db_port, db_name, user, password, gene_d
-                        return version.group()
+        if file_format == "owl":
+            for event, elem in ET.iterparse(fh, events=("start", "end")):
+                if event == "start" and elem.tag == "{http://www.w3.org/1999/02/22-rdf-syntax-ns#}RDF":
+                    for i in elem.iter():
+                        if i.tag == "{http://www.w3.org/2002/07/owl#}versionIRI":
+                            resource = i.get("{http://www.w3.org/1999/02/22-rdf-syntax-ns#}resource")
+                            version = re.search("[0-9]+\\-[0-9]+\\-[0-9]+", resource)
+                            return version.group()
+        else:
+            for line in fh:
+                if line.startswith("#"):
+                    data_row = line.split(",")
+                    version = data_row[0].replace("##", "")
 
     return version
 
+def split_mondo_file(file):
+    Path("tmp").mkdir(parents=True, exist_ok=True)
+    output_file = "tmp/mondo_"
+    count = 1
+    header = None
+
+    with open(file, 'r') as f2:
+        data = f2.read()
+
+        for chunk in re.split(r"\s*\<\!\-\-*", data):
+            if count == 1:
+                header = chunk
+                count = count + 1
+            else:
+                with open(output_file+str(count)+".owl", "w") as wr:
+                    wr.write(header)
+                    wr.write("\n<!-- "+chunk)
+                    if "</rdf:RDF>" not in chunk:
+                        wr.write("\n</rdf:RDF>")
+                    count = count + 1
 
 def main():
     parser = argparse.ArgumentParser(description="")
-    parser.add_argument("--host", required=False, help="Ensembl core database host")
-    parser.add_argument("--port", required=False, help="Ensembl core host port")
-    parser.add_argument("--database", required=False, help="Ensembl core database name")
-    parser.add_argument("--user", required=False, help="Username")
-    parser.add_argument("--password", default='', help="Password (default: '')")
-    parser.add_argument("--g2p_host", required=True, help="G2P database host")
-    parser.add_argument("--g2p_port", required=True, help="G2P host port")
-    parser.add_argument("--g2p_database", required=True, help="G2P database name")
-    parser.add_argument("--g2p_user", required=True, help="Username")
-    parser.add_argument("--g2p_password", default='', help="Password (default: '')")
+    parser.add_argument("--config", required=True, help="Config file with details to Ensembl and G2P databases")
     parser.add_argument("--omim", default=1, help="Import OMIM gene-disease data")
     parser.add_argument("--mondo", default=1, help="Import MONDO gene-disease data")
-    parser.add_argument("--mondo_file", default='', help="MONDO owl file")
+    parser.add_argument("--mondo_file", default='', help="MONDO file in the format: owl or csv")
+    parser.add_argument("--update", default=0, help="Run data update. By default, the script imports all data from scratch")
```

Comment:
Suggesting we use update as a flag instead `action="store_true" `might be the best way so if --update is called then it is set to True

Reply:
Applied this logic to all options that can be saved as booleans.